gtfu/src/main/java/gtfu/Calendar.java

```diff
@@ -71,4 +83,27 @@ public String toShortList() {
 
         return sb.toString();
     }
-}
\ No newline at end of file
+
+    public String toArrayString() {
+        StringBuffer sb = new StringBuffer();
+
+        /*
+        0 -> 2
+        1 -> 3
+        2 -> 4
+        3 -> 5
+        4 -> 6
+        5 -> 7
+        6 -> 1
+        */
+
+        sb.append("[");
+        for (int i=0; i<7; i++) {
+            int n = ((i + 1) % 7 + 1);
+            sb.append(days[n] == 0 ? "0" : "1");
+            sb.append(i < 6 ? ", " : "");
```

Comment:
May I suggest `DAYS_PER_WEEK - 1` here and `DAYS_PER_WEEK` in the two occurrences above? Should probably go with the other constants in `Time.java`.